LICENSE.md

```diff
@@ -0,0 +1,428 @@
+# Attribution-NonCommercial-ShareAlike 4.0 International
+
+Creative Commons Corporation ("Creative Commons") is not a law firm and
+does not provide legal services or legal advice. Distribution of
+Creative Commons public licenses does not create a lawyer-client or
+other relationship. Creative Commons makes its licenses and related
+information available on an "as-is" basis. Creative Commons gives no
+warranties regarding its licenses, any material licensed under their
+terms and conditions, or any related information. Creative Commons
+disclaims all liability for damages resulting from their use to the
+fullest extent possible.
+
+Using Creative Commons Public Licenses
+
+Creative Commons public licenses provide a standard set of terms and
+conditions that creators and other rights holders may use to share
+original works of authorship and other material subject to copyright
+and certain other rights specified in the public license below. The
+following considerations are for informational purposes only, are not
+exhaustive, and do not form part of our licenses.
+
+**Considerations for licensors:**  
+Our public licenses are  
+intended for use by those authorized to give the public  
+permission to use material in ways otherwise restricted by  
+copyright and certain other rights. Our licenses are  
+irrevocable. Licensors should read and understand the terms  
+and conditions of the license they choose before applying it.  
+Licensors should also secure all rights necessary before  
+applying our licenses so that the public can reuse the  
+material as expected. Licensors should clearly mark any  
+material not subject to the license. This includes other CC-licensed  
+material, or material used under an exception or  
+limitation to copyright. More considerations for licensors:  
+ wiki.creativecommons.org/Considerations_for_licensors
+
+**Considerations for the public:**  
+By using one of our public
+licenses, a licensor grants the public permission to use the  
+licensed material under specified terms and conditions. If  
+the licensor's permission is not necessary for any reason--for  
+example, because of any applicable exception or limitation to  
+copyright--then that use is not regulated by the license. Our  
+licenses grant only permissions under copyright and certain  
+other rights that a licensor has authority to grant. Use of  
+the licensed material may still be restricted for other  
+reasons, including because others have copyright or other  
+rights in the material. A licensor may make special requests,  
+such as asking that all changes be marked or described.  
+Although not required by our licenses, you are encouraged to  
+respect those requests where reasonable. More_considerations  
+for the public:  
+ wiki.creativecommons.org/Considerations_for_licensees
```

Comment:
Those still look "cut" as they have two trailing spaces, whereas the rest of the document does not.